labgrid/util/marker.py

```diff
@@ -2,5 +2,8 @@
 import string
 
 
+# Remove RID to avoid markers containing substrings like ERROR, FAIL, WARN, INFO or DEBUG
+MARKER_POOL = ''.join(c for c in string.ascii_uppercase if c not in 'RID')
```

Comment:
```suggestion
MARKER_POOL = tuple(c for c in string.ascii_uppercase if c not in 'RID')
```

Reply:
Yes it can be any sequence for `choice()`